### PR TITLE
[Containerapp] az containerapp service: Fix deprecate version

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

The old deprecate version is too close to remove the related commands, so the CI linter failed as following, we need to set the version to a bigger one, to give the buffer time to remove the old command in the next extension version:
```
-  FAIL - HIGH severity: expired_command_group
    Command-Group: `containerapp service kafka` - Deprecated command group is expired and should be removed.
    Command-Group: `containerapp service mariadb` - Deprecated command group is expired and should be removed.
    Command-Group: `containerapp service postgres` - Deprecated command group is expired and should be removed.
    Command-Group: `containerapp service qdrant` - Deprecated command group is expired and should be removed.
    Command-Group: `containerapp service redis` - Deprecated command group is expired and should be removed.
    Command-Group: `containerapp service` - Deprecated command group is expired and should be removed.
```

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Containerapp

**Review by bgashirabake — APPROVED** [review]

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Release History
 upcoming
 ++++++
 * 'az containerapp add-on' : support for add-on milvus create and delete commands
-* [Breaking Change] 'az containerapp service': deprecate command from Azure CLI version 2.56.1
+* [Breaking Change] 'az containerapp service': deprecate command from Azure CLI version 2.59.0
```
Comment (Juliehzl) [inline src/containerapp/HISTORY.rst:8]: 2.58.1

File: src/containerapp/azext_containerapp/commands.py [hunk 2]
```diff
@@ -50,45 +50,45 @@ def load_command_table(self, _):
...
+    with self.command_group('containerapp service postgres', deprecate_info=self.deprecate(redirect='containerapp add-on postgres', expiration='2.58.1', hide=True)) as g:
         g.custom_command('create', 'create_postgres_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_postgres_service', confirmation=True, supports_no_wait=True)
 
     with self.command_group('containerapp add-on postgres') as g:
         g.custom_command('create', 'create_postgres_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_postgres_service', confirmation=True, supports_no_wait=True)
 
-    with self.command_group('containerapp service kafka', deprecate_info=self.deprecate(redirect='containerapp add-on kafka', expiration='2.56.1', hide=True)) as g:
+    with self.command_group('containerapp service kafka', deprecate_info=self.deprecate(redirect='containerapp add-on kafka', expiration='2.59.0', hide=True)) as g:
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/commands.py:75]: 2.58.1

File: src/containerapp/azext_containerapp/commands.py [hunk 3]
```diff
@@ -50,45 +50,45 @@ def load_command_table(self, _):
...
         g.custom_command('create', 'create_kafka_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_kafka_service', confirmation=True, supports_no_wait=True)
 
     with self.command_group('containerapp add-on kafka') as g:
         g.custom_command('create', 'create_kafka_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_kafka_service', confirmation=True, supports_no_wait=True)
 
-    with self.command_group('containerapp service mariadb', deprecate_info=self.deprecate(redirect='containerapp add-on mariadb', expiration='2.56.1', hide=True)) as g:
+    with self.command_group('containerapp service mariadb', deprecate_info=self.deprecate(redirect='containerapp add-on mariadb', expiration='2.59.0', hide=True)) as g:
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/commands.py:83]: same for the following

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 4]
```diff
@@ -931,8 +931,8 @@ def test_containerapp_dev_service_binding_customized_keys_yaml_e2e(self, resourc
                 JMESPathCheck('properties.template.serviceBinds[1].customizedKeys', None),
             ])
 
-        self.cmd('containerapp service redis delete -g {} -n {} --yes'.format(env_rg, redis_ca_name), expect_failure=False)
-        self.cmd('containerapp service postgres delete -g {} -n {} --yes'.format(env_rg, postgres_ca_name), expect_failure=False)
+        self.cmd('containerapp add-on redis delete -g {} -n {} --yes'.format(env_rg, redis_ca_name), expect_failure=False)
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:934]: if you set target version to 2.59.0, then the test cases should keep until azurecli upgrades to 2.59.0

Comment (Greedygre) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:1133]: Dup tests.

**Review by Juliehzl — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Pull request merged**